config/hub/manifests/odr/kustomization.yaml

```diff
@@ -6,6 +6,9 @@ resources:
 
 patches:
 - patch: |-
+    - op: add
+      path: /metadata/annotations/operatorframework.io~1suggested-namespace
+      value: openshift-dr-cluster
```

Comment:
Is the value for the namespace correct here?

Reply:
Are you thinking `odr`? We decided that all namespaces start with `openshift` and hence we do not want to shorten it. Is there another problem you see?

Reply:
If the namespace is "openshift-dr-cluster" for both the hub and cluster operators then this is correct. No other issue.

Reply:
Ack! Changing this to `openshift-dr-system` namespace across both hub and cluster bundles.